tslint.json

```diff
@@ -85,6 +85,7 @@
     "strict-string-expressions": false,
     "whitespace": false,
     "no-async-without-await": false,
-    "invalid-void": false
+    "invalid-void": false,
+    "max-file-line-count": [false]
```

Comment:
Default value is 1000 line. I don't suppose we intend to break up this file any time soon so I just disabled the lint.

Reply:
TBH, 1000 is already wayyy too high. Following the last PR, I did create a ticket to split things up, see [CA-955](https://reach5.atlassian.net/browse/CA-955).